app/controllers/internal_controller.rb

```diff
@@ -62,4 +62,10 @@ def list_registrations
 
     render json: registrations
   end
+
+  def registrations_for_user
+    user_id = params.require(:user_id)
+    registrations = Registration.where(user_id: user_id).map { |x| { competition_id: x.competition_id, status: x.competing_status } }
+    render json: { registrations: registrations }
```

Comment:
Is it necessary to have this top-level `registrations` key? The other method that lists registrations for a competition just does `return json: registrations` directly, and I feel like that's the more concise way to do it.

Reply:
good point, I implemented it without that key on the monolith as well